auto_nag/user_activity.py

```diff
@@ -0,0 +1,95 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import datetime, timedelta
+from enum import Enum
+
+from libmozdata.bugzilla import Bugzilla, BugzillaUser
+
+from auto_nag import utils
+
+DEFULT_ACTIVITY_WEEKS = 26
```

Comment:
nit: typo

Reply:
Done in 57b46c3c937504c988dddd0a52f138ff5c9a93b8